.github/workflows/check.yml

```diff
@@ -53,25 +53,23 @@ jobs:
 
   fmt:
     runs-on: ubuntu-latest
-    name: nightly / fmt
+    name: stable / fmt
```

Comment:
rustfmt on nightly is a concious decisions because of our of module level `use`

Reply:
Is that particularly important to us? A dependency on `nightly` for day to day devex tasks should be a high bar.

Reply:
Only really important for embassy HALs, really.